exercise3/src/camera_demo/CMakeLists.txt

```diff
@@ -0,0 +1,81 @@
+cmake_minimum_required(VERSION 3.8)
+project(camera_demo)
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+# uncomment the following section in order to fill in
+# further dependencies manually.
+# find_package(<dependency> REQUIRED)
+
+find_package(rclcpp REQUIRED)
+find_package(sensor_msgs REQUIRED)
+find_package(cv_bridge REQUIRED)
+find_package(image_transport REQUIRED)
+find_package(OpenCV REQUIRED)
```

Comment:
Do we actually need these? Can we use a fake image as @JanStaschulat suggested (like an array of `int[2048]` for example)? This will allow us to cut down on dependencies which will improve stability and also reduce the size of the docker and raspberry pi images we need to distribute to people